haskell/test/Lorentz/Contracts/Test/FA2.hs

```diff
@@ -76,6 +76,21 @@ fa2Spec fa2Originate = do
           validate . Right $
             lExpectViewConsumerStorage consumer [balanceExpected]
 
+    it "allows zero transfer from non-existent accounts" $ integrationalTestExpectation $ do
+      -- Tests transactions are applied in order
+      -- Update balances exactly
+        let originationParams = addOperator (wallet1, commonOperator) $ defaultOriginationParams
+                { opPermissionsDescriptor = permissionDescriptorOperatorTransferAllowed }
+        withOriginated fa2Originate originationParams $ \fa2contract -> do
+          let
+            transfers = constructTransfersFromSender (#from_ .! wallet1)
```

Comment:
In this case it's better to use `constructSingleTransfer`

Reply:
Changed